### DISCUSSION TITLE
Error Relation Manager View Not Found

### DISCUSSION THREAD

**Discussion by lutfifirdaus** [filamentphp/filament#15099] [Help]
### Package

Panel builder

### Package Version

v3.2

### How can we help you?

Hello, I have problem when I try create new relation manager, its return this error, which I think the relation manager's view is inherit from package right? anyone know how I can fix this?

![image](https://github.com/user-attachments/assets/16e9768c-da13-4ea6-ae81-4795ef605ff7)

**Comment by leandrocfe** [comment]
Did you create it using [this command](https://filamentphp.com/docs/3.x/panels/resources/relation-managers#creating-a-relation-manager) ?

If you are optimizing filament, run `php artisan filament:optimize-clear`
